sklearn/metrics/scorer.py

```diff
@@ -313,12 +322,12 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 
 # Standard regression scores
 r2_scorer = make_scorer(r2_score)
-mean_squared_error_scorer = make_scorer(mean_squared_error,
-                                        greater_is_better=False)
-mean_absolute_error_scorer = make_scorer(mean_absolute_error,
-                                         greater_is_better=False)
-median_absolute_error_scorer = make_scorer(median_absolute_error,
-                                           greater_is_better=False)
```

Comment:
Maybe we could add a `deprecation_message` option to `make_scorer` to make sure that the deprecation warning is raised when the user actually call the scorer instead of raising the warning in `get_scorer` that only covers a subset of the public API.

```python
neg_median_absolute_error_scorer = make_scorer(median_absolute_error,
                                               greater_is_better=False)
median_absolute_error_scorer = make_scorer(
    median_absolute_error,
    greater_is_better=False,
    deprecation_msg='Scoring method median_absolute_error was renamed to '
                    'neg_median_absolute_error in version 0.18 and will be '
                    'removed in 0.20.'
)
```